variables.tf

```diff
@@ -116,6 +116,18 @@ variable "db_maintenance_window" {
   default     = "wed:03:00-wed:04:00"
 }
 
+variable "db_apply_immediately" {
+  description = "Specifies whether any database modifications are applied immediately, or during the next maintenance window."
+  type        = bool
+  default     = true
```

Comment:
default should be `false` IMO. This is an open source module, others might not want this behaviour.

Reply:
ok

Reply:
I agree on this